app/konnect/dev-portal/customization/custom-domain.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Add a Custom Domain
+no_version: true
+---
+
+All Dev Portals have an assigned, generated Dev Portal URL. To further customize your Dev Portal, you can add a custom domain. When set up properly, users can access both the assigned URL and the custom URL.
+
+## Prerequisites
+
+* Access to your organization's {{site.konnect_short_name}} Admin UI.
+* A domain with access to configure that domain's CNAME. Kong does not offer any custom domains as a service.
+* A signed certificate, which comes with a private key.
```

Comment:
Do you need to indicate the signed certificate is an X.509 certificate, as called out in the steps below?

Reply:
This page looks great. My only other suggestion is the Customization Overview needs an edit to include features other than appearance settings, such as the customizing the domain and public portal.

Reply:
This is a standard certificate. Sure.

Reply:
The Customization Overview is out of scope for this specific PR but I can address that at a later time.

Reply:
While the Customization Overview is out of scope for this PR, it needs to be updated and link to this topic because the current overview has a note to contact Kong Support to customize the Portal URL.